src/objectTranslations/Class_Period__c-ca.objectTranslation

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- Class Period --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- Class Period --></value>
+    </caseValues>
+    <fields>
+        <help><!-- The education organization associated with this period. For example, a school within a district. --></help>
```

Comment:
All object translations probably need to be re-retrieved.

Reply:
I must be doing something wrong (and sorry I re-requested review, may not have been ready)